notebooks/tutorials/visualization/blender/landlab_to_blender.ipynb

```diff
@@ -0,0 +1,362 @@
+    "\n",
+    "0 = place the viewpoint at the position of the camera\n",
+    "\n",
+    "1, 3, 7 = snap view along y-axis / x-axis / z-axis (respectively) (notice how the guide axes in upper right change)\n",
+    "\n",
+    "5 = toggle between perspective and orthographic view\n",
+    "\n",
+    "9 = rotate view 180 degrees\n",
+    "\n",
+    "#### On the \"main\" part of the keyboard\n",
+    "\n",
+    "Tab = toggle between object mode and edit mode\n",
+    "\n",
+    "1 = toggle vertex select mode\n",
+    "\n",
+    "2 = toggle edge select mode\n",
+    "\n",
+    "3 = toggle face select mode\n",
+    "\n",
+    "a = select all (alt-a to unselect all)\n",
+    "\n",
+    "r = toggle rotate mode. You can rotate the whole object or an individually selected element; holding down the `x` key while you rotate will limit rotation to the x-axis; holding Shift-x will limit rotation to the other two axes. The same functionality applies with `y` and `z`. Holding the control key while rotating causes rotation to snap to finite increments (e.g., 15 degrees). Left-click keeps the rotation, and right-click discards it.\n",
+    "\n",
+    "s = toggle scale mode. Limiting scale to x, y, or z axes works the same as with rotation.\n",
+    "\n",
+    "g = toggle grab mode, which allows you to \"grab\" the object and move it around. As above, motion can be limited to just one or two axes.\n",
+    "\n",
+    "z = hold this key down to select the view mode. Options are *Solid* (the default), *Wireframe*, *Material preview*, and *Rendered*. \n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Applying Subdivision\n",
+    "\n",
+    "Blender can automatically subdivide a model, adding additional vertices and faces to increase the level of detail. For a Landlab-generated model, this can help add smoothness to the image. (Be careful: too much subdivision and the model can become too big for the computer to handle.)\n",
+    "\n",
+    "To apply subdivision:\n",
+    "\n",
+    "1. From the column of icons just to the (lower) right of the main view pane, select the one that looks like a blue wrench.\n",
+    "\n",
+    "2. From the drop-down menu *Add Modifier* select *Subdivision Surface*\n",
+    "\n",
+    "3. The *Levels Viewport* allows you to choose a number of recursion levels for the model as it appears in the viewport. Try setting it to 2. (The *Render* box sets the recursion levels for the rendered image; often 3D designers will keep the viewport subdivision smaller than the rendered version for the purpose of speed in object editing)\n",
+    "\n",
+    "Here's a version of our terrain model with 2 levels of subdivision, with the lighting on viewport shading also changed to a brownish hue (hint: see the buttons and drop-down arrow on the upper-right of the viewport):\n",
+    "\n",
+    "<div>\n",
+    "    <img src=\"assets/muddy_terrain.png\">\n",
```

Comment:
again, screenshot not appearing

Reply:
Oops, assets folder didn't get added because it's .gitignored. Shrunk figures and embedded them in notebook insteadl.